hedyorg/hedy
# Fix assign list highlighting from level 16 #5355

Fixes #5355 
Highlighting now works correctly for list definitions in level 16 and up.

**How to test**
- Automated tests are added to cover the change. To test this manually, run Hedy locally and check that the following programs have correct highlighting in level 16 and up:
```
list = []
list = [1]
list = [50.1]
list = ['test']
```
```
# even though these are not a correct programs, the highlighting should not break
q = 1
list = [q]
list = [q,q,q]
list = [1, '2', q]
```
